kubernetes_asyncio/client/rest.py

```diff
@@ -78,7 +78,8 @@ def __init__(self, configuration, pools_size=4, maxsize=None):
 
         # https pool manager
         self.pool_manager = aiohttp.ClientSession(
-            connector=connector
+            connector=connector,
+            trust_env=True
```

Comment:
This file is generated by https://github.com/OpenAPITools/openapi-generator and you have to modify the generator first.

Reply:
ok, I did it yesterday, let's see what happens :) thanks

https://github.com/OpenAPITools/openapi-generator/pull/11171/files

Reply:
the PR on openapi-generator was approved, sould this PR be approved also? Thanks!